src/index.html

```diff
@@ -11,7 +11,185 @@
+          max="100"
+        >
+        <br>
+        <label for="date">Full date of birth:</label>
+        <input
+      class="duo"
+      type="date"
+      id="date"
+      name="date"
+        >
+        <br>
+        <label for="box">I accept the term of the agreement</label>
+        <input
+      type="checkbox"
+      id="box"
+      name="box"
+        >
+        <br>
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+        <label for="adress">E-mail:</label>
+        <input
+      type="email"
+      id="adress"
+      class="duo"
+      name="adress"
+      placeholder="email@example.com"
+        >
+        <br>
+        <label for="secret">Password:</label>
+        <input
+      type="password"
+      id="secret"
+      name="secret"
+      required
+      autocomplete="off"
+      minlength="8"
+      maxlength="15"
+        >
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <div class="duo">
+          <label for="cat">Do you love cats?</label>
+          <input
+           id="cat"
+           type="radio"
+           name="pet"
+           value="yes"
+          >
+        Yes
+          <input
+           type="radio"
+          name="pet"
+          value="No"
+          >
+        No
+        </div>
+        <div class="duo">
+          <label for="color">What is your favorite color?</label>
+          <input
+          type="color"
+          id="color"
+          name="color"
+          >
+        </div>
+
+        <div class="duo">
+          <label for="time">What time do you go to bed?</label>
+          <input
+          type="time"
+          id="time"
+          name="time"
+          >
+        </div>
+
+        <div class="duo">
+          <label for="brand">What are your favorite brand of cars?</label>
+          <select
+           name="car"
+           id="brand"
+           required
+           multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </div>
+
+        <div>
+          <label for="review">How do you rate our work?</label>
+          <input type="range"
+           id="review"
+           name="rating"
+           step="1"
+           max="0"
+           min="0"
+          >
+        </div>
+
+      </fieldset>
+
+      <fieldset>
+        <legend>Additional info</legend>
+
+        <div class="quads">
+          <label for="comments">Comments:</label>
+
+          <textarea name="write"
+          id="comments"
+          autocomplete="off"
+          required
+         ></textarea>
+        </div>
+
+        <label for="approve">Would you recommend us?</label>
+        <select
+        name="answer"
+        id="approve"
+        required
+        >
+          <option
+          value="Choose"
+          disabled
+          selected
+            >
+          yes
```

Comment:
Why do you need 2 yes options? 
<img width="277" alt="image" src="https://github.com/mate-academy/layout_html-form/assets/99403198/024f9ac4-7e7b-4653-9574-6f743e83d146">